### PR TITLE
DM-25479: Reuse python ingest client code for filesystem

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: rootfs/ingest/python/qserv/ingest.py [hunk 1]
```diff
@@ -176,11 +176,11 @@ def get_transactions(self):
...
         for transaction_id in self.get_transactions():
             success = False
-            self._close_transaction(self.chunk_meta.database,
-                                    transaction_id, success)
+            self._close_transaction(transaction_id, success)
             _LOG.info("Abort transaction: %s", transaction_id)
 
-    def _start_transaction(self, database):
+    def _start_transaction(self):
+        database = self.chunk_meta.database
```
Comment (fjammes) [inline rootfs/ingest/python/qserv/ingest.py:183]: :+1:

File: rootfs/ingest/python/qserv/util.py [hunk 2]
```diff
@@ -66,9 +67,17 @@ def download_file(base_url, filename):
 def json_get(base_url, filename):
     """Load json file at a given URL
     """
-    file_url = urllib.parse.urljoin(base_url, filename)
-    r = requests.get(file_url)
-    return r.json()
+    url = urllib.parse.urlsplit(base_url, scheme="file")
```
Comment (fjammes) [inline rootfs/ingest/python/qserv/util.py:70]: :+1:

**Review by fjammes — APPROVED** [review]
LGTM, thanks for your support on this code!
I'll try to fix the CI ASAP, the error certainly comes from my side.

**Comment by hsinfang** [comment]
Thank you for the review!

**Pull request merged**